src/frontends/onnx/docs/how_to_add_op.md

```diff
@@ -0,0 +1,2 @@
+# How to add an new operation
+TBD
```

Comment:
Do we have a ticket to cover all TBDs in future?

Reply:
yes, just as summary:
93001 - how to add a new op
93002 - ONNX FE tests
94113 - class diagram

btw, a great idea of component doc ;) I have a plan to fill sub pages content after FF